papis/commands/default.py

```diff
@@ -231,26 +245,25 @@ def run(verbose: bool,
         import atexit
         atexit.register(generate_profile_writing_function(profiler, profile))
 
-    import colorama
     if _disable_color(color):
         # Turn off colorama (strip escape sequences from the output)
         colorama.init(strip=True)
     else:
         colorama.init()
 
-    log_format = (colorama.Fore.YELLOW
+    log_format = ("["
                   + "%(levelname)s"
-                  + ":"
+                  + "] "
                   + colorama.Fore.GREEN
                   + "%(name)s"
                   + colorama.Style.RESET_ALL
-                  + ":"
+                  + " "
                   + "%(message)s"
                   )
```

Comment:
I also changed the log format from
```
LEVEL:module:module:Some message over here
```
to
```
[LEVEL] module:module Some message over here
```
i.e. put the level in brackets and add another space before the message. 

What do you think? I'm not super sold on either version, but was looking at it and wanted to bring it up at least. I'll revert it back to the previous version if no one has any preferences for now.

Reply:
I'm wondering why not just keep the ".", rather than transforming it into ":", then we could have the message be `[LEVEL] module.module: Some message over here` which would retain the colon as a separator between the message and the stuff before.

Reply:
Yeah, that looks good! Thanks for the suggestion!

For reference, default `dmesg` does
```
[timestamp] submodule: Some message
```
and `journalctl` does
```
timestamp host submodule: Some message
``` 
(without any other brackets) so this should be nice and in line with some other common tools.